index.html

```diff
@@ -1,91 +1,154 @@
-<!DOCTYPE html>
+<!DOCTYPE html> 
 <html>
 
 <head>
+
     <link rel="stylesheet" href="./style.css">
+
 </head>
 
+
 <body>
     <header id="header">
         <img src='https://cdn.freecodecamp.org/testable-projects-fcc/images/product-landing-page-logo.png'
             id='header-img'>
-        <nav id="nav-bar">
```

Comment:
It would be nice if you could add a class to your nav tag.

Reply:
Actually i don't understand what you are saying